<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1292</width>
    <height>843</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="submitPB">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>220</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Sumbit</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>87</width>
      <height>37</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="statusTip">
     <string>Select the required faculty</string>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="maxVisibleItems">
     <number>50</number>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
    </property>
    <property name="duplicatesEnabled">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>NVS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MDB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NPS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LL</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelx">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>330</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>166</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string>Faculty Code</string>
    </property>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>40</y>
      <width>124</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string>Time slot</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>100</y>
      <width>165</width>
      <height>37</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="statusTip">
     <string>Select the time slot</string>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="maxVisibleItems">
     <number>50</number>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
    </property>
    <property name="duplicatesEnabled">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>Home Room</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8:00-9:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9:00-10:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10:15-11:15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11:15-12:15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1:00-2:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2:00-3:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3:00-4:00</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labely">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>330</y>
      <width>171</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1292</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
